File: 作业10-反转数组/java/Solution.java
public class Solution {

	public static int[] reverseArray(int[] nums) {
		int end = nums.length - 1;
		int start = 0;
		while (start < end) {
			int temp = nums[start];
			nums[start] = nums[end];
			nums[end] = temp;
			start++;
			end--;
		}
	}
}

File: 作业10-反转数组/php/Solution.php
<?php

class Solution {
	function reverseArray($nums) {
		$start = 0;
		$end = count($nums) - 1;
		while ($start < $end) {
			$tmp = $nums[$start];
			$nums[$start] = $nums[$end];
			$nums[$end] = $tmp;
			$start++;
			$end--;
		}
		return $nums;
	}
}

echo json_encode((new Solution())->reverseArray([1, 2, 3, 4, 5])) . PHP_EOL;